example/scripts/tests-parallel.sh

```diff
@@ -14,7 +14,7 @@ testOutputDir=scripts/test_output
 rm -r -f $testOutputDir
 mkdir -p $testOutputDir
 
-echo "" | ../node_modules/.bin/parallel -j 0 sh scripts/run_test.sh  $testOutputDir ::: $TESTS
+echo "" | ../node_modules/.bin/parallel -j 0 --timeout 3000 sh scripts/run_test.sh  $testOutputDir ::: $TESTS
```

Comment:
Does 3000 = 5 mins, not 50 minutes?

Reply:
Yep, thank you. I had been going back and forth and playing around with different values.